sufia.gemspec

```diff
@@ -9,12 +9,14 @@ Gem::Specification.new do |gem|
   gem.homepage      = "http://github.com/projecthydra/sufia"
 
   gem.files         = `git ls-files | grep -v ^sufia-models`.split($\)
+  gem.bindir        = 'exe'
   gem.executables   = gem.files.grep(%r{^bin/}).map{ |f| File.basename(f) }
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.name          = "sufia"
   gem.require_paths = ["lib"]
   gem.version       = version
   gem.license       = 'Apache2'
+  gem.executables   = gem.files.grep(%r{^exe/}) { |f| File.basename(f) }
```

Comment:
@mjgiarlo @cam156 "exe" is the new location for gem executables so that it keeps it separate from the other like rake and rails. Are we saying we don't want to adopt the new convention or is there another reason we're sticking with the old way? http://bundler.io/blog/2015/03/20/moving-bins-to-exe.html